client/src/app/home/home.component.html

```diff
@@ -1,6 +1,6 @@
 <div fxLayout="row">
   <div fxFlex fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10">
-    <mat-card class="home-card">
+    <mat-card appearance="outlined" class="home-card">
```

Comment:
Was this (and all the other additions of `appearance="outlined"`) needed as part of the update, or was it seen as an improvement to the interface? (Without running the previous version next to this one, it's hard to see what [if anything] this does.)

Reply:
It did look a little bit like you could not discern the location of the edges of the cards when we removed (and had not replaced) flex-layout, so I suspect this explains the reasoning behind this change.